### PR TITLE
Boss alien

### PR DESCRIPTION
**Pull request by egedebruin**
There is now every third round a boss alien wave with one boss alien which shoots multiple bullets at once. The sprite used for now is large alien.

### CODE REVIEW

**Comment by jochem725** [comment]
Looks nice. There are a few check style warnings remaining.
To solve the design for extension errors, I would suggest a refactor in which Alien has two subclasses, DefaultAlien and BossAlien. DefaultAlien has subclasses NormalAlien, HealthAlien etc.

In this way the move and shootBullet methods can be made final in DefaultAlien and BossAlien, solving the design for extension error making Alien SOLID compliant again.

**Comment by egedebruin** [comment]
Good solution, I have implemented it in the last commit.

**Comment by bryanvanwijk** [comment]
I think it would be better to increase the bulletChance for the BossAlien to make it more difficult.

**Comment by jochem725** [comment]
I've fixed the final 2 checkstyle issues remaining for you, I agree with merging!

File: src/main/java/spaceinvaders/group_22/unit/Alien.java [hunk 1]
```diff
@@ -43,11 +45,13 @@ public Alien(final double x, final double y) {
...
-		bullet.setVelY(velocity);
-		return bullet;
-	}
+	public abstract ArrayList<Bullet> shootBullet(final double velocity);
+//		ArrayList<Bullet> list = new ArrayList<Bullet>();
+//		Bullet bullet = new AlienBullet(this.getXCoor(), this.getYCoor());
+//		bullet.setVelY(velocity);
+//		list.add(bullet);
+//		return list;
+//	}
```
Comment (bryanvanwijk) [inline src/main/java/spaceinvaders/group_22/unit/Alien.java]: Why is this code commented? if it isn't needed anymore deleted it.

**Comment by egedebruin** [comment]
I see that that one is outdated, Jochem already deleted it.
I have increased the bulletChance to the same as a shootAlien.

**Comment by bryanvanwijk** [comment]
I think it can be merged now

**Pull request merged**